src/objects.js

```diff
@@ -1,41 +1,51 @@
 const createPerson = (name, age) => {
-  // your code here
+  return {name, age};
 };
 
 const getName = object => {
-  // your code here
+  return object.name;
 };
 
 const getProperty = (property, object) => {
-  // your code here
+  return object[property];
 };
 
 const hasProperty = (property, object) => {
-  // your code here
+  return object[property] > "";
 };
 
 const isOver65 = person => {
-  // your code here
+  return person.age > 65;
 };
 
 const getAges = people => {
-  // your code here
+  return people.filter(x=>x.name).map(x=>x.age);
```

Comment:
You dont need to use the `filter` method here too, can just use the map part of this:
```
  return people.map(person => person.age);
```
Also its good to use variable names that can be easily understood at first glance. Only person I can think of with the name X is one of elon musks kids